lib/util/http/index.ts

```diff
@@ -245,6 +245,11 @@ export class Http<Opts extends HttpOptions = HttpOptions> {
       method,
       responseType: 'json',
     };
+    // signal that we expect a json response
+    opts.headers = {
+      Accept: 'application/json',
```

Comment:
```suggestion
      accept: 'application/json',
```

should be lowercased

Reply:
![image](https://user-images.githubusercontent.com/1798109/205041623-75fb8c60-30f4-4a2d-9da1-6ff1c87e2344.png)